$MirOS: ports/emulators/qemu/patches/patch-audio_audio_c,v 1.2 2005/05/03 19:30:03 tg Exp $
--- audio/audio.c.orig	Fri Nov 11 00:04:19 2005
+++ audio/audio.c	Thu Nov 17 14:05:23 2005
@@ -23,6 +23,10 @@
  */
 #include "vl.h"
 
+#ifndef INT16_MAX
+#define INT16_MAX		(32767)
+#endif
+
 #define AUDIO_CAP "audio"
 #include "audio_int.h"
 
